public/js/app.js

```diff
@@ -231,7 +231,26 @@ var runExperiment = function() {
   return x();
 };
 
+// Run the experiment then post to server
 runExperiment().then(function() {
```

Comment:
Should we decouple the function here, so it is easier to read, as well being more modular.